weather_mv/README.md

```diff
@@ -25,7 +25,8 @@ _Common options_:
 
 * `-i, --uris`: (required) URI prefix matching input netcdf objects. Ex: gs://ecmwf/era5/era5-2015-""
 * `-o, --output_table`: (required) Full name of destination BigQuery table. Ex: my_project.my_dataset.my_table
-* `-v, --variables`:  Target variables for the BigQuery schema. Default: will import all data variables as columns.
+* `-v, --variables`:  Target variables (or coordinates) for the BigQuery schema. Default: will import all data variables
```

Comment:
s/coordinates/non-indexed coordinates/ ?
(or a better word)

Reply:
Yeah, I can see how leaving it this vague would be confusing. If users supply variables, we hard-code adding lat/lng and time. Further, the indexed-coordinates should already be added at the time we check for them in `to_rows()`.  So, in some sense, we allow adding both coordinates and non-indexed coordinates. 

Given all this, I think that coordinates in this text is accurate. WDYT?